### PR TITLE
create handleEmailSubmit function

### PR DESCRIPTION
**Pull request by upluke**
### Pull Request Summary

Create a Send receipt button that accepts a list of items and generates a report listing all donations and sends an email to the test email via SES.

### Pull Request Checklist

- [x] Code is neat, readable, and works
- [x] Comments are appropriate
- [x] The commit messages follows our [guidelines](https://h4i.notion.site/Conventional-Commits-593452ad1179489399ad3bd696ef772a)
- [x] The developer name is specified
- [x] The summary is completed
- [x] Ask for a review in communication channels

### Reviewer Checklist - PULL REQUEST REVIEWER ONLY

IMPORTANT: The rest of the sections in this checklist should only be filled out by authorized pull request reviewers. If you are the individual contributor, do not fill out the rest of the fields or check the boxes.

## Reviewer: Full Name

NOTE: Pull requests should only be merged when all boxes are checked.

- [ ] Assign yourself as reviewer if not assigned
- [ ] The reviewer name is specified
- [ ] The code is fully reviewed
- [ ] Meaningful feedback is given
- [ ] Let developer know a review has been made

### CODE REVIEW

File: src/components/AddItemButton.tsx [hunk 1]
```diff
@@ -69,11 +69,33 @@ function AddItemButton() {
...
+    fetch("https://xgjrtv69b5.execute-api.us-west-2.amazonaws.com/sendEmail", {
+      mode: "no-cors",
+      method: "POST",
+      headers: {
+        Accept: "application/json",
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        senderName: "wilshire-health",
+        senderEmail: "lukeforadream@gmail.com",
```
[the remark this answers is not in the document]
Comment (upluke) [reply src/components/AddItemButton.tsx]: Updated!

File: src/components/AddItemButton.tsx [hunk 2]
```diff
@@ -69,11 +69,34 @@ function AddItemButton() {
...
+  };
+
   return (
     <div>
       <button type="button" onClick={handleClick}>
         Submit item
       </button>
+
+      <button type="button" onClick={handleEmailSubmit}>
+        Send email
```
[the remark this answers is not in the document]
Comment (upluke) [reply src/components/AddItemButton.tsx:99]: Updated!

File: src/components/AddItemButton.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (upluke) [reply src/components/AddItemButton.tsx]: Screen Shot 2023-05-16 at 11 26 06 PM

**Review by jlaksana — APPROVED** [review]
Got it to work for me! Thanks for making that request. Proceed with task #42

**Pull request merged**